src/index.ts

```diff
@@ -35,10 +49,26 @@ export function runInContext(
   context: Context,
   options: Partial<IOptions> = {}
 ): Promise<Result> {
+  function getFirstLine(theProgram: Program) {
+    if (theProgram.body[0] && theProgram.body[0].type === 'ExpressionStatement') {
+      const firstLineOfProgram = theProgram.body[0] as ExpressionStatement
+      const theDirective = (firstLineOfProgram as Directive).directive
+      if (theDirective) {
+        return theDirective
+      }
+    }
+
+    return undefined
+  }
   const theOptions: IOptions = { ...DEFAULT_OPTIONS, ...options }
   context.errors = []
   const program = parse(code, context)
   if (program) {
+    if (getFirstLine(program) === 'enable verbose') {
+      verboseErrors = true
```

Comment:
This kind of `if true then set to true else set to false` pattern is not as good as simply writing `verboseErrors = getFirstLine(program) === 'enable verbose'`

On the other hand, as commented above, might be good to extract this function out for testing